internal/nas/handler.go

```diff
@@ -10,39 +10,49 @@ import (
 	"github.com/free5gc/nas"
 )
 
-func HandleNAS(ue *amf_context.RanUe, procedureCode int64, nasPdu []byte, initialMessage bool) {
+func HandleNAS(ranUe *amf_context.RanUe, procedureCode int64, nasPdu []byte, initialMessage bool) {
 	amfSelf := amf_context.GetSelf()
 
-	if ue == nil {
+	if ranUe == nil {
 		logger.NasLog.Error("RanUe is nil")
 		return
 	}
 
 	if nasPdu == nil {
-		ue.Log.Error("nasPdu is nil")
+		ranUe.Log.Error("nasPdu is nil")
 		return
 	}
 
-	if ue.AmfUe == nil {
-		ue.AmfUe = amfSelf.NewAmfUe("")
-		gmm_common.AttachRanUeToAmfUeAndReleaseOldIfAny(ue.AmfUe, ue)
+	if ranUe.AmfUe == nil {
+		if ranUe.FindAmfUe != nil && !ranUe.FindAmfUe.CmConnect(ranUe.Ran.AnType) {
+			// models.CmState_IDLE
+			gmm_common.ClearHoldingRanUe(ranUe.FindAmfUe.RanUe[ranUe.Ran.AnType])
```

Comment:
` !ranUe.FindAmfUe.CmConnect(ranUe.Ran.AnType)` implies that ranUe.FinAmfUe didn't have underlying ranUe, right? 
Shouldn't ClearHoldingRanUe(ranUe.FindAmfUe.RanUe[ranUe.Ran.AnType]) do nothing?

Reply:
The `ClearHoldingRanUe(ranUe.FindAmfUe.RanUe[ranUe.Ran.AnType])` do noting. 
This line had been remove in the latest commit.